backend/server/routers/courses.py

```diff
@@ -122,7 +122,7 @@ def getAllUnlocked(userData: UserData):
     that they have already completed"""
 
     coursesState = {}
-    user = User(fixUserData(userData.dict()))
+    user = User(fixUserData(userData.dict())) if type(userData) != User else userData
```

Comment:
Added this so i could get pass in the user object

Reply:
oh btw i think you forgot to add the test file

Reply:
Added